src/sentry/static/sentry/app/actionCreators/deployPreview.tsx

```diff
@@ -46,5 +47,6 @@ export function displayExperimentalSpaAlert() {
     ),
     type: 'warning',
     neverExpire: true,
+    noDuplicates: true,
```

Comment:
Does this filename still make sense since now it's doing double duty for the production API dev stuff

Reply:
Hmm yeah it could use a better name, any ideas?

Reply:
Maybe `spaMode`

Reply:
or `actionCreators/spaHelpers`